dwi_ml/data/dataset/batch_sampler.py

```diff
@@ -0,0 +1,166 @@
+import logging
+
+from torch.utils.data import Dataset, Sampler
+import numpy as np
```

Comment:
I would try to stick to PEP to order imports unless there are conflicts. Improves readability.